### ISSUE TITLE
Scissors bug?

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1999]
https://clips.twitch.tv/ObservantElatedInternCoolCat-wG3qshLKf0kee1rG

not sure why other bros don't shot tears, they gave up on this run waaay too early

**Comment by Lonrite** [comment]
![image](https://github.com/UnderscoreKilburn/repentance-issues/assets/74597482/97547d24-838c-4a24-9400-bed55968d435)

also if you have homing only your head gains it, not the body

idk if it's intentional or no but it just makes me wanna stop use that item entirely with homing

**Comment by Rchardon** [comment]
Of course, scissors don't use your tears, like Shard of Glass. Imagine with ipecac if it worked that way
